### DISCUSSION TITLE
Italian Support

### DISCUSSION THREAD

**Discussion by ilgrandeanonimo** [community/community#47622] [Mobile]
If possible I would like to help translating the app in Italian.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.99.0 (90728769)
Device: iPhone 11 (iOS 16.3.1)
</details>

**Comment by candyho** [comment]
Thanks for your feedback! I'll add this feedback to our internal localization issue for tracking! Hopefully we could support more languages in the near future.
